ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis.ecf

```diff
@@ -29,21 +21,21 @@ export model=gfs
 ############################################################
 # Load modules
 ############################################################
-module load lsf/${lsf_ver}
-module load impi/${impi_ver}
-module load CFP/${CFP_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load cfp/${cfp_ver}
 module load python/${python_ver}
-module load NetCDF-parallel/${netcdf_parallel_ver}
-module load HDF5-parallel/${hdf5_parallel_ver}
-module load crtm/${crtm_ver}
+module load libjpeg/${libjpeg_ver}
+module load hdf5/${hdf5_ver}
+module load netcdf/${netcdf_ver}
 module load grib_util/${grib_util_ver}
+module load crtm/${crtm_ver}
 
 module list
 
 #############################################################
 # WCOSS environment settings
 #############################################################
-export KMP_AFFINITY=scatter
 export OMP_NUM_THREADS=$threads
 export FORT_BUFFERED=true
 export OMP_NUM_THREADS_CY=28
```

Comment:
Is `OMP_NUM_THREADS_CY=28` appropriate for WCOSS2?  `config.resources.nco.static` is setting this to `npe_node_max`. `28` is the value for `WCOSS_Dell_P3` and was hard-wired here.

Reply:
The same is the case in `jgfs_atmos_analysis.ecf`.

Reply:
Set to 128

Reply:
> Set to 128

Are you sure, the value from this variable set here `OMP_NUM_THREADS_CY` is being used?
I see https://github.com/NOAA-EMC/GSI/blob/a9561f4219e5a69ac1af250081ae027084c40c3b/scripts/exglobal_atmos_analysis.sh#L1017 setting the value based on `NTHREADS_CYCLE` right before entering `CYCLESH`